helm-chart/renku/values.yaml

```diff
@@ -74,6 +74,17 @@ global:
       ## specify which dataset should be used in Jena
       dataset:
 
+  ## If you want to use an external PostgreSQL instance not deployed by Renku,
+  ## specify the hostname/IP address, admin username, password and maintenance database
+  ## of your external PostgreSQL instance and set externalPostgresql.enabled to true.
+  ## postgresql.enabled must also be set to false to use an external PostgreSQL instance.
+  externalPostgresql:
+    enabled: false
```

Comment:
it feels like this option and the one below should really be merged:
```
postgresql:
  embedded: true
  # If false an external Postgres must be defined [...]
```

Reply:
I was trying to think of a good way to do this. We need the `postgresql.enabled` option to be able to disable the deployment of the PostgreSQL chart, but at the same time, I don't want to define connection options for an external Postgresql deployment under the `postgresql` chart values. What do you think?

Reply:
Instead of `postgresql.enabled` can we use `global.postgresql.embedded`? Essentially moving it up one level.